go/store/nbs/table_set.go

```diff
@@ -115,6 +115,35 @@ func (ts tableSet) hasMany(addrs []hasRecord) (bool, error) {
 	return f(ts.upstream)
 }
 
+func (ts tableSet) hasManyInSources(srcs []hash.Hash, addrs []hasRecord) (remaining bool, err error) {
```

Comment:
Most functions on tableSet are documented in the chunkReader interface. This one is a one off though, so adding gDocs seems appropriate. I'm not wild about this function name being the same as the NomsBlockStore function, especially since they are different interfaces. m2c.

Reply:
Commented. I was mirroring the |hasMany| symmetry on NomsBlockStore/tableSet, but I agree it can be a bit confusing. Not revisiting the naming for now though...